android/src/main/kotlin/com/amplitude/amplitude_flutter/AmplitudeFlutterPlugin.kt

```diff
@@ -127,6 +154,31 @@ class AmplitudeFlutterPlugin : FlutterPlugin, MethodCallHandler {
         }
     }
 
+    private fun trackAppLifecycleAndDeepLinkEvents(appLifecycles: Boolean, deepLinks: Boolean) {
+        amplitude.isBuilt.invokeOnCompletion { exception ->
+            if (exception != null) {
+                println("isBuilt computation failed with exception: $exception")
+            } else {
+                val utils = DefaultEventUtils(amplitude)
```

Comment:
Does it make sense to build the utils object also during init?

Reply:
I'd like to keep it in `trackAppLifecycleAndDeepLinkEvents()`:
- `utils` is only used here
- if here is an exception when `invokeOnCompletion`, we don't need to build `utils`
wdyt?

Reply:
Sure. No strong preference. We can keep it here.